Rubberduck.Core/UI/UnitTesting/Commands/AddTestMethodExpectedErrorCommand.cs

```diff
@@ -116,6 +118,7 @@ protected override void OnExecute(object parameter)
             _state.OnParseRequested(this);
         }
 
+        [Obsolete] // Duplicates AddTestMethodCommand#GetNextTestMethodName
```

Comment:
Shouldn't that comment be the `string` argument of the `ObsoleteAttribute`? If the duplication is necessary, a comment could say _why_ ;-)